docker/README.md

```diff
@@ -0,0 +1,50 @@
+# How to use docker to spin up WebSiteOne
+
+## Prerequisites
+
+In order to run this container you'll need docker installation
+* [Windows](https://docs.docker.com/docker-for-windows/)
+* [OS X](https://docs.docker.com/docker-for-mac/)
+* [Linux](https://docs.docker.com/linux/started/)
+
+## Setup the project
+Follow the instruction at [Project README.md](../README.md) to set up the configs for the project
```

Comment:
We probably don't want to point them to the current README since its for local development and involves configuring dbs locally @brotich

Reply:
@brotich but if they are following docker installation, they don't need to follow those steps, right?  so we shouldn't link to it in docker documentation is what I mean.

Reply:
we still need the steps above are still needed for now as the setup is still tied to the steps mentioned above. Once the above is resolved, we can use the docker setup as the main developer flow. 

Notes: The steps needed above are to set up the `.example` files

Reply:
I will pull this in now.  we should probably investigate this further.